wolfssl/wolfcrypt/port/Espressif/esp_crt_bundle.h

```diff
@@ -0,0 +1,233 @@
+/* esp_crt_bundle.h
+ *
+ * Copyright (C) 2006-2024 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#ifdef HAVE_CONFIG_H
+    #include <config.h>
+#endif
+
+#ifndef __ESP_CRT_BUNDLE_wolfssl_LIB_H__
+
+#define __ESP_CRT_BUNDLE_wolfssl_LIB_H__
+
+/* This file is typically NOT directly used by applications utilizing the
+ * wolfSSL libraries. It is used when the wolfssl libary component is configured
+ * to be utilized by the Espressif ESP-IDF, specifically the esp-tls layer.
+ *
+ * See:
+ * https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/protocols/esp_tls.html
```

Comment:
Maybe this link could go in the documentation or manual rather than in the main wolfssl bundle. The site looks hosted in Poland which is okayish for some users, but the certificate on the site has CN listed as the country.

Reply:
I found 59 instances of `docs.espressif.com` in 49 files. 

I'll remove or change the links in this PR. 

Ok to have a separate PR for the remainder or would you prefer they are included here?

Reply:
Update: a total of 112 instances in 72 files that reference `espressif.com` in a comment.

Reply:
Update[2]: a total of 206 instances in 102 files, if including `README.md` files in addition to `*.c` and `*.h` files.

Reply:
separate PR's is ok